.vscode/launch.json

```diff
@@ -35,6 +35,8 @@
             },
             "linux": {
                 "env": {
+                    // platform-specific launch config overwrites (rather than merges with) existing config, so need to specify CARGO_MANIFEST_DIR again
```

Comment:
Usually json does not allow comments. Can you remove this please?

Reply:
In this case VSCode [officially supports JSON comments](https://code.visualstudio.com/docs/languages/json#_json-with-comments) in its `settings.json`, `tasks.json` and `launch.json` files (and of course, this launch config also does actually work too).

Happy to remove it though if you like - let me know.

Reply:
I had thought so, but couldn't find it in the docs. Thank you for the link. In that case it is of course OK.